pkg/webhook/core.oam.dev/v1alpha2/applicationconfiguration/validating_handler.go

```diff
@@ -187,14 +194,14 @@ func ValidateTraitAppliableToWorkloadFn(_ context.Context, v ValidatingAppConfig
 				if strings.HasPrefix(applyTo, "*.") && workloadGroup == applyTo[2:] {
 					continue ValidateApplyTo
 				}
-				if workloadType == applyTo ||
-					workloadDefRefName == applyTo {
+				if workloadDefRefName == applyTo ||
+					workloadDefName == applyTo {
```

Comment:
we should rename them and add explaination here:
```
// according to OAM convention, Spec.Reference.Name in workloadDefinition is CRD name
var crdName = workloadDefRefName
// according to OAM convention, name of workloadDefinition is the workload type.
var workloadTypeName = workloadDefName
```

Reply:
got it.